doto-frontend/src/components/ModalContent.js

```diff
@@ -72,6 +73,13 @@ const ModalContent = props => {
         }
     };
 
+    const handleEarliestChange = date => {
+        if (date > new Date()) {
+            setEarliestDate(date);
+        } else {
+            setEarliestDate("invalid beans");
```

Comment:
Would `"invalid beans"` be an appropriate error message here? I noticed down later the error message is `"Date must be after now"` inside of `<KeyboardDateTimePicker/>`

I have noticed that it was the same as the above function, maybe we could improve on this?
(Would be unreasonable to carry over coding conventions that do not make sense.)